packages/server/lib/util/args.js

```diff
@@ -448,6 +450,10 @@ module.exports = {
       }
     }
 
+    if (autoCancelAfterFailures && !((typeof autoCancelAfterFailures === 'number' && autoCancelAfterFailures % 1 === 0) || autoCancelAfterFailures === false)) {
```

Comment:
why do we need `autoCancelAfterFailures % 1 === 0`? Should we only care if the number is positive?

Reply:
We only want to handle integer values - and this way we prevent float values

Reply:
ah I see, was there issues with `Number.isInteger()`?

Reply:
gonna update it!